Sources/Core/aws-sdk-swift.AWSClientRuntime/Package.swift

```diff
@@ -40,8 +44,8 @@ let package = Package(
             name: "AWSClientRuntimeTests",
             dependencies: [
                 "AWSClientRuntime",
-                .product(name: "ClientRuntime", package: "aws-sdk-swift.smithy-swift"),
-                .product(name: "SmithyTestUtil", package: "aws-sdk-swift.smithy-swift"),
+                .product(name: "ClientRuntime", package: "smithy-swift"),
+                .product(name: "SmithyTestUtil", package: "smithy-swift"),
                 .product(name: "AWSSDKCommon", package: "aws-sdk-swift.AWSSDKCommon"),
```

Comment:
Below, you will see that all the AWSClientRuntime files have been moved to a new directory location.